controllers/genreController.js

```diff
@@ -0,0 +1,25 @@
+const { Genre } = require('../models');
+
+/**
+ * Serves an array of movie genres
+ * @return  genres
+ */
+module.exports.readAll = async (req, res) => {
+  const allGenres = await Genre.find({});
+
+  if (res) res.json(allGenres);
+  else return allGenres;
+};
+
+module.exports.readOne = async (req, res) => {
+  const foundGenre = await Genre.findById(req.params.id);
+  res.json(foundGenre);
+};
+
+module.exports.readOneByKey = key =>
+  Genre.findOne({ 'external_ids.tmdb_id': key }).exec();
```

Comment:
Make this one line with no break after the arrow. 
```suggestion
const readOneByKey = key => Genre.findOne({ 'external_ids.tmdb_id': key }).exec();
```

Reply:
Prettier automatically separates the function into two lines for readability, since it's too long (maximum is 80 characters per line).